apiary.apib

```diff
@@ -240,14 +245,19 @@ and details in the response body:
 }
 ```
 ## Partial response
-To relieve network throughput, fields in json responses can be filtered out by providing query parameter ```fields``` with comma separated list of fields that you are interested in.
+To relieve network throughput, fields in json responses can be filtered out by providing query parameter ```fields``` with comma separated list of fields that you are interested in. Nested objects can be filtered
+with `.` notation. There is simple filter to include all fields of nested object with `.*`
+
+Examples:
 
-There are two restrictions:
-* Some fields cannot be filtered out. Typically boolean values or sume numbers. There is no list of this values
-* Only 1st level fields can be filtered. Filtering inside nested json objects is not possible.
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,url```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,url,borrowerRelatedInfo.*```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,photos.url```
+- ```GET https://api.zonky.cz/loans/marketplace?fields=id,photos.*```
 
-Example:
-Request: ```GET http://api.zonky/loans/marketplace?fields=id,url```
+Request:
+
+```GET https://api.zonky.cz/loans/marketplace?fields=id,url```
```

Comment:
Could you please add an example of the filtering with `.` notation?

Reply:
You mean response example? Because request examples are in previous section

Reply:
You're right. So it's ok, sorry.